FirebaseAI/Sources/AILog.swift

```diff
@@ -138,6 +139,17 @@ enum AILog {
     log(level: .debug, code: code, message)
   }
 
+  @available(iOS 15.0, macOS 12.0, macCatalyst 15.0, tvOS 15.0, watchOS 8.0, *)
+  static func makeInternalError(message: String, code: MessageCode) -> GenerateContentError {
+    let error = GenerateContentError.internalError(underlying: NSError(
+      domain: "com.google.generative-ai",
+      code: -1,
```

Comment:
Thanks for adding the malformedURL handling! Would you mind changing this to something like this? I had been meaning to update `"com.google.generative-ai"` to `"com.google.firebase.firebaseai"`.

```suggestion
      domain: "\(Constants.baseErrorDomain).Internal",
      code: code.rawValue,
```